packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -207,15 +207,13 @@ export const selectClass = style([
 
 export const chevronIconClass = style([
   sprinkles({
-    height: '$9',
     marginRight: '$2',
     position: 'absolute',
     right: '$1',
     color: '$gray40',
-    display: 'inline-flex',
-    alignItems: 'center',
   }),
   {
+    top: '20%',
```

Comment:
What was the reason for removing the flex properties and making this 20% from the top all the time? Right now it looks like the select component grows to fit it's container, but the icon doesn't. So this could still result in the the icon not being centered